pallets/nfts/src/weights.rs

```diff
@@ -243,7 +243,9 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	/// Storage: `Nfts::Item` (r:1 w:1)
 	/// Proof: `Nfts::Item` (`max_values`: None, `max_size`: Some(861), added: 3336, mode: `MaxEncodedLen`)
 	/// Storage: `Nfts::AccountBalance` (r:2 w:2)
-	/// Proof: `Nfts::AccountBalance` (`max_values`: None, `max_size`: Some(72), added: 2547, mode: `MaxEncodedLen`)
+	/// Proof: `Nfts::AccountBalance` (`max_values`: None, `max_size`: Some(120), added: 2595, mode: `MaxEncodedLen`)
+	/// Storage: `System::Account` (r:1 w:1)
```

Comment:
Just confirming that this is due to the deposit being charged from dest when they have an account with sufficient free balance for the collection balance deposit? It would be useful to have a review comment on these changes explaining why they are happening.

Reply:
- `r: 1`: Read the balance of the `dest` account with `T::Currency::can_reserve` to identify the depositor of the call. 
- `w: 1`: Reserve from the `depositor` account. Either the `caller` or the `dest` account.